www/nginx/src/opnsense/service/templates/OPNsense/Nginx/http.conf

```
@@ -130,6 +130,7 @@ server {
         root /var/etc/acme-client/challenges;
     }
 {% endif %}
+{% if server.disable_bot_protection is defined and server.disable_bot_protection == '0' %}
```

can you try that one - should be more fail safe
```suggestion
{% if server.disable_bot_protection is not defined or server.disable_bot_protection != '1' %}
```

---

yours would currently suppress the block if the option is missing.

---

Sure